builtin/plugin/backend.go

```diff
@@ -152,14 +152,16 @@ func (b *PluginBackend) HandleRequest(ctx context.Context, req *logical.Request)
 			err := b.startBackend(ctx)
```

Comment:
I agree that there is a bug here -- we need to call Initialize() every time the plugin gets loaded.  I do think however that we want to call it explictly at mount time also, as we are doing now.  I will make a change so that the call to Initialize() happens in startBackend()

Reply:
I've posted a potential fix for this, please have a look.